pkg/rhttp/datatx/manager/simple/simple.go

```diff
@@ -87,8 +101,16 @@ func (m *manager) Handler(fs storage.FS) (http.Handler, error) {
 
 		switch r.Method {
 		case "GET", "HEAD":
+			downloadsActive.Add(1)
```

Comment:
only count downloads for GET, not HEAD?

Reply:
I was considering that, but thought that HEAD requests would be short enough to not matter. I'll do it for GET so that the graphs are less jittery.